src/main/java/org/opentripplanner/routing/algorithm/astar/strategies/MultiTargetTerminationStrategy.java

```diff
@@ -10,31 +10,31 @@
 
 /**
  * A termination strategy that terminates after multiple targets have been reached.
- * 
```

Comment:
This class is only used in a test. Should it be moved to the `test` folder or removed entirely?